src/libxtp/qmpackages/orca.cc

```diff
@@ -312,6 +313,8 @@ bool Orca::WriteShellScript() {
   }
   shell_file << _settings.get("executable") << " " << _input_file_name << " > "
              << _log_file_name << endl;  //" 2> run.error" << endl;
+  std::string base_name = _mo_file_name.substr(0, _mo_file_name.size() - 4);
+  shell_file << "orca_2mkl " << base_name << " -molden" << endl;
```

Comment:
should this be part of the execute script or of the parse part? Because if it is part of the parse, this change would even be backwards compatible.

Reply:
Smart, I could make it part of the parse part

Reply:
This is more difficult than expected. There is a problem with testing the results. I can add this to the ParseMOsFile, but then I can't test the MO file parsing, because in the test we need access to a shell with orca loaded in the path. I thought to be smart and split the ParseMOsFile function in two (one with the shell script and one with the rest of the parsing) and only test the second part in the unit test, but with this virtual class stuff that can't be done, because the two separate functions aren't present in the qmpackage class. If you have any ideas on how to work around that issue I would be glad to hear. As an alternative we could not go for the backwards compatibility but raise an error if someone tries to do it the old way saying that they can convert their .gbw file with the orca_2mkl tool manually. I'll implement the latter for now.

Reply:
Yes that is a good solution.